pkgs/overlay.nix

```diff
@@ -434,6 +436,12 @@ in {
     ];
   });
 
+  poetry = nixpkgs-22_11.poetry;
+  poetry2nix = (super.lib.debug.traceVal nixpkgs-22_11.poetry2nix);
```

Comment:
Do we still need the tracing? If yes, why?

Reply:
no we don't, good catch!